terms.cform

```diff
@@ -114,6 +114,14 @@
 
         \No Other Licenses\  Apart from the licenses in {Intellectual Property}, this agreement does not license or assign any <Intellectual Property Right>.
 
+        \Dual Licensing\  If <Developer> makes all or any part of the <Software> available under any <Public License>:
+
+            \\  The licenses given under this agreement are in addition to any <Public License> for the <Software>.
+
+            \\  <Customer> need not comply with the terms of any <Public License> for actions covered by a license under this agreement.
```

Comment:
In lay terms, does this mean you can do anything that the Public license restricts, as long as it is not also restricted by this Paid license?

Reply:
It's not clear enough. I'll try to revise.